### PR TITLE
Update 2013-06-10-fix-typo-in-documentation.md

### PR DESCRIPTION
**Pull request by mjbeyeler**

### CODE REVIEW

File: content/en/2013-06-10-fix-typo-in-documentation.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ date: '2013-06-10'
 slug: fix-typo-in-documentation
 ---
 
-> Update (2017/08/16): It is much easier to fix typos on my web pages now. There is a menu item "Edit this page" on each page. You can edit the source of a page if you find any mistakes, and propose a correction through a Github pull request. You don't need to use any command-line tools. You should not follow the instructions in the post below any more (the **knitr** documentation has been moved to a different repo).
+> Update (2017/08/16): It is much easier to fix typos on my web pages now. There is a menu item "Edit this page" on each page. You can edit the source of a page if you find any mistakes, and propose a correction through a Github pull request. You don't need to use any command-line tools. You should not follow the instructions in the post below any more (the **knitr** documentation has been moved to a different repo). Great idea! ;)
```
Comment (yihui) [inline content/en/2013-06-10-fix-typo-in-documentation.md:7]: Thanks! I guess you don't really mean a pull request (but a comment instead) here?

**Comment by MichaelBeyeler** [comment]
Yes! I've just been trying things out. xD

File: content/en/2013-06-10-fix-typo-in-documentation.md (hunk 1 above)
Comment (mjbeyeler) [reply content/en/2013-06-10-fix-typo-in-documentation.md:7]: Yes! I've just been trying things out. xD

Comment (yihui) [reply content/en/2013-06-10-fix-typo-in-documentation.md:7]: Very cool. Thanks, and congratulations on successfully sending a pull request!! 👍

**Pull request closed without merging**